frontend/src/Layout/StudentDashBoard/StudentDashBoard.jsx

```
@@ -0,0 +1,239 @@
+                                tabIndex={0}
+                                className="card card-compact dropdown-content bg-gray-900 text-white z-[10] mt-3 w-72 shadow-xl rounded-lg">
+                                <div className="card-body p-4">
+                                    <div className="flex justify-between items-center border-b border-gray-700 pb-2">
+                                        <span className="text-lg font-bold">📢 Notifications</span>
+                                        <button className="text-sm text-gray-400 hover:text-white">Clear all</button>
+                                    </div>
+
+                                    {/* Notification Items */}
+                                    <div className="mt-2 space-y-3">
+                                        <button className="flex items-start p-3 bg-gray-800 hover:bg-gray-700 rounded-lg w-full text-left">
+                                            <div className="mr-3 text-blue-400 text-xl">📅</div>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)